src/Randomizer.Shared/Models/Seed.cs

```diff
@@ -1,19 +1,21 @@
 ﻿using System.Collections.Generic;
+using System;
+using System.ComponentModel.DataAnnotations.Schema;
 
-namespace Randomizer.Shared.Models {
-
-    public class Seed {
+namespace Randomizer.Shared.Models
+{
+    public class Seed
+    {
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
         public int Id { get; set; }
-        public string Guid { get; set; }
-        public string Mode { get; set; }
-        public string SeedNumber { get; set; }
-        public string Spoiler { get; set; }
-        public string GameName { get; set; }
-        public string GameVersion { get; set; }
-        public string GameId { get; set; }
-        public string Hash { get; set; }
-        public int Players { get; set; }
-        public List<World> Worlds { get; set; }
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public DateTime Date { get; set; }
```

Comment:
Should probably be DateTimeOffset, and I don't know if having the date as ID is a good idea. Maybe just stick to an int ID for this?